proposed_projects/DockerHub.md

```diff
@@ -2,7 +2,19 @@
 
 ## Description
 
-Build various dockerHub development stacks, such as MEAN, LAMP, Tomcat, based on OpenSUSE
+This project will produce a suite of Docker images for OpenSUSE and ClefOS (a CentOS clone for Z).
+
+If you go to Dockerhub and look at https://cloud.docker.com/u/clefos/repository/list you’ll see all the existing builds for Z. These are all based on a CentOS clone called ClefOS. The source for most of these (excluding the OpenShift and Source-to-Image images) may be found at https://github.com/nealef/docker. 
```

Comment:
@nealef Hi, this link `https://cloud.docker.com/u/clefos/repository/list` gives a 404. It should be changed to just `https://cloud.docker.com/u/clefos/` this url lists all the images under the user clefos

Reply:
@palash25 `https://cloud.docker.com/u/clefos/` also gives 404. I think it should it `https://hub.docker.com/u/clefos`

Reply:
yup. sorry about that. :sweat_smile: